xbmc/video/VideoDimensions.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+/*
+ *      Copyright (C) 2005-2013 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include <string>
+#include <map>
+#include <vector>
+#include <utility>
+
+typedef std::pair<std::pair<int, int>, std::pair<int, int>> Boundary;
```

Comment:
I'd prefer a struct with more obvious names over a combination of `std::pair` inside `std::pair`. You'd have a struct with `min` and `max` members and one with `width` and `height` members of the previous struct.